src/occupation.jl

```diff
@@ -175,3 +175,13 @@ function compute_fermi_level(basis::PlaneWaveBasis{T}, eigenvalues, ::FermiZeroT
 
     εF
 end
+
+"""
+Check that all orbitals are fully occupied.
+"""
+function check_full_occupation(basis::PlaneWaveBasis, occupation)
+    filled_occ = filled_occupation(basis.model)
+    for occ_k = occupation
```

Comment:
`occk` and `for occk in `

Reply:
I think `for x = ` is akward but it was changed in many places in https://github.com/JuliaMolSim/DFTK.jl/pull/920 so I kept using it.